DynamicQueryBuilder/DynamicQueryAttributeGlobalConfig.cs

```diff
@@ -0,0 +1,22 @@
+using DynamicQueryBuilder.Models.Enums;
+
+namespace DynamicQueryBuilder
+{
+    public static class DynamicQueryAttributeGlobalConfig
```

Comment:
As much as i like the simple development aspect of this static context, this probably will come with its prices like thread safety and all. Instead, i would advise to use dependency injection through the ActionContext and get our settings from there. [Here](https://github.com/oplog/DynamicQueryBuilder/blob/811ac9e5135e36242cd08f6f38d0500a105e429b/DynamicQueryBuilder/DynamicQueryAttribute.cs#L53) is an example of that.